### ISSUE TITLE
Categorical Distribution Loss calculation

### ISSUE THREAD

**Issue by bertoloti** [ajgupta93/d4pg-pytorch#2]
In line 217 of ddpg.py file the loss is calculated as the mean of the multiplication of the projected distribution and the log of the q distribution. But according to the D4PG paper, equation 10, the loss in terms of the cross-entropy is calculated different. Can you detail why you choose to calculate the loss as you did? Thanks

**Comment by GoingMyWay** [comment]
@bertoloti   Have you figured out the code of D4PG?
